setup.py

```diff
@@ -142,10 +142,9 @@ def _set_manifest_path(manifest_dir: str, aggregate: bool = False, mapping: Mapp
         package_to_install = _PACKAGE_MAPPING.get(_PACKAGE_NAME, "lightning")
         if package_to_install == "lightning":
             # merge all requirements files
-            assistant._load_aggregate_requirements(_PATH_REQUIRE, _FREEZE_REQUIREMENTS)
-        if package_to_install == "lightning" or _PACKAGE_NAME in _MIRROR_PACKAGE_REVERSED:  # install everything
-            # replace imports and copy the code
-            assistant.create_mirror_package(_PATH_SRC, _PACKAGE_MAPPING, reverse=_MIRROR_PACKAGE_REVERSED)
+            assistant._load_aggregate_requirements(_PATH_REQUIRE, _FREEZE_REQUIREMENTS)  # install everything
+        # replace imports and copy the code
+        assistant.create_mirror_package(_PATH_SRC, _PACKAGE_MAPPING, reverse=_MIRROR_PACKAGE_REVERSED)
```

Comment:
this is the place we create mirror fabric also for building PL package, as mention in https://github.com/Lightning-AI/lightning/pull/16589#issuecomment-1412007110